FAQ.md

```diff
@@ -8,8 +8,7 @@ You can opt-in for the beta build by clicking on this link: https://play.google.
 - STEP 4: You have to copy or move the file into a location on the SD Card: whatever_sdcard/RetroMusic/lyrics/ and paste inside your ".lrc" file in there.
 
 #### Q: Why isn't the artist's image downloading?
-STEP 1: Settings -> Images -> Auto-download artist images -> Always
-STEP 2: Open the artists tab and wait some time for the images to download.
+Last.fm has disabled the download of artist's images for the time being, whether functionality for this will be restored in future is uncertain.
```

Comment:
Here is an alternative https://developer.spotify.com/documentation/web-api/reference/artists/get-artist/